store/app.py

```diff
@@ -1,10 +1,74 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+
 app = Flask(__name__)
 
-@app.route("/users/<username>")
-def hello(username):
-        return "Hello {}".format(username)
+products = [
+        {
+            'id': 1,
+            'size': 7,
+            'price': 99,
+            'country': 'Brazil'
+        },
+        {
+            'id': 2,
+            'size': 4,
+            'price': 77,
+            'country': 'China'
+        },
+        {
+            'id': 3,
+            'size': 9,
+            'price': 45,
+            'country': 'EUA'
+        }
+]
+
+@app.route('/')
+@app.route('/api/v1/products')
+def home():
+    return jsonify(products)
+
+
+@app.route('/api/v1/product/<int:product_id>')
+def get_product_id(product_id):
+    for product in products:
+        if product['id'] == product_id:
+            return jsonify(products[product_id - 1]), 200
+    return jsonify({'Error':'Not Found'}), 404
+
+
+@app.route('/api/v1/product/<int:product_id>', methods=['PUT'])
+def change_country(product_id):
```

Comment:
the name of the function is `change_country` and the parameter is `product_id`? weird right?